### PR TITLE
tests/nightly: Re-enable satellite regression test on nightly composes

### PR DESCRIPTION
**Pull request by jrusz**
I might have made a mistake when I tested this last time. Trying again.

This pull request includes:

- [ ] adequate testing for the new functionality or fixed issue
- [ ] adequate documentation informing people about the change such as
  - [ ] submit a PR for the [guides](https://github.com/osbuild/guides) repository if this PR changed any behavior described there: https://www.osbuild.org/guides/

### CODE REVIEW

**Comment by jrusz** [comment]
Nightly pipeline run with latest changes: https://gitlab.com/osbuild/ci/osbuild-composer/-/pipelines/416531988

**Comment by jrusz** [comment]
Regression test in nightly pipeline passed https://gitlab.com/osbuild/ci/osbuild-composer/-/jobs/1820611505

File: tools/define-compose-url.sh [hunk 1]
```diff
@@ -8,16 +8,16 @@ if [[ $ID != rhel ]]; then
 fi
 
 if [[ $ID == rhel && ${VERSION_ID%.*} == 8 ]]; then
-  COMPOSE_ID=$(curl -L http://download.devel.redhat.com/rhel-8/nightly/RHEL-8/latest-finished-RHEL-"${VERSION_ID}"/COMPOSE_ID)
+  COMPOSE_ID=$(curl -L http://download.eng.bos.redhat.com/rhel-8/nightly/RHEL-8/latest-finished-RHEL-"${VERSION_ID}"/COMPOSE_ID)
```
Comment (atodorov) [inline tools/define-compose-url.sh:11]: What is the reason for this change? 

AFAIK .devel. will redirect to the nearest mirror which could be eng.bos or anything else depending on where we provision.

**Review by atodorov — COMMENTED** [review]
Looks good. Added one comment to understand the reason for changing some of the URLs.

File: tools/define-compose-url.sh (hunk 1 above)
Comment (jrusz) [reply tools/define-compose-url.sh:11]: >  What is the reason for this change?

There can be a time where other mirrors are not synced with bos yet (which I believe has happened when investigating the failures on nightlies) and by hard-coding this to bos we can be sure the compose is always there. But now that I think about it I'm not so sure about the benefit. If failure due to what I just said happens we can just restart the pipeline at a later time and if we see failures because of this we can always make this change later.. So would you prefer to keep devel there?

**Review by atodorov — APPROVED** [review]
